.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 'Feature Request'
+about: 'Request a new feature or suggest a change'
+title: '[suggestion] SHORT DESCRIPTION'
```

Comment:
```suggestion
title: '[feature] SHORT DESCRIPTION'
```

Reply:
what about the https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md#issues? The description says "suggestion"

Reply:
and also talk about "feature" that matches the label. I've changed it already.